tracks/python/exercises/perfect-numbers/mentoring.md

```diff
@@ -48,6 +48,6 @@ def classify(number):
 ### Points to call out
 
 Students unfamiliar with `generator expressions` might write: `aliquot = sum([item for item in range(1, number) if number % item == 0])`.
-Note this first creates a `list` of factors in memory by iterating over the entire range, then iterates once more over the `list` to `sum()` its values.
-This is inefficent for both memory and processing time.
-Dropping the `[]` drops `list` creation and allows `sum()` to lazily process a `generator expression`, wich only requires a single iteration and a smaller memory footprint.
+Note that this first creates a `list` of factors in memory by iterating over the entire range, then iterating its values once more over the `list` to `sum()`.
```

Comment:
The original version of this line is correct as-is.

Reply:
Changed to original version.